docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.react.mdx

```diff
@@ -160,6 +160,18 @@ This is a known problem when using the `jsdom` library (a dependency of Jest) wi
 
 Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. Please see the Next.js [documentation](https://nextjs.org/docs/getting-started/react-essentials#third-party-packages) on how you could achieve this.
 
+### Next 13.4+: `Module not found errors`
+When you use Connected components, you may see the follow errors from `aws-crt` and `encoding` on the server:
```

Comment:
The user will definitely see these errors correct? Might be misunderstanding but can't think of a scenario where the user would not see these warnings when using server components?

Reply:
+1 we should say they "will see" instead of "may see". If my understanding is correct, this actually happens even using client components since Next.js will still server render the component and rehydrate on the frontend by default. Also, this error is only seen in the terminal, not the browser console, we may want to clarify that.

Reply:
You are right. I will change the wording